pageserver/src/tenant/timeline.rs

```diff
@@ -1780,7 +1780,7 @@ impl Timeline {
             // need to return something
             Ok(0)
         });
-        let timer = if up_to_lsn == self.initdb_lsn {
+        let timer = if Some(up_to_lsn) == self.current_logical_size.initial_part_end {
```

Comment:
as comment says,  `initial_part_end` could be absent for freshly created timelines.
(not sure what exactly this "could be" means)

Am I getting it right, that this change will break  `init_logical_size_histo` for new timelines?

Reply:
Aah I did not consider this.

Reply:
new attempt: [5cfc5f3](https://github.com/neondatabase/neon/pull/3594/commits/5cfc5f398c86f4af4e8a26ba88af7dba659f8989)

Reply:
"Freshly crated timeline" does not seem hard to create, could some unit tests help detection here?

Reply:
Good idea on the unit test. I've abused this CurrentLogicalSize enough to warrant some refactoring :)